src/App.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import './App.css';
+import './randomQuote.ts';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <p>"To be honest you guy don't smell as bad as I thought you would"</p>
+        <randomQuote />
```

Comment:
have also tried with this wrapped in p tag

Reply:
`<p>{randomQuote()}</p>`

Reply:
This does not work. `'randomQuote' is not defined no-undef`

Reply:
Because I needed this import:
```javascript
import randomQuote from './randomQuote.ts';
```